apps/plotly_examples/src/components/DeclarativeChart.tsx

```diff
@@ -65,7 +65,7 @@ type DataType =
   | 'annotations';
 
 const dataTypeRanges = {
-  'general': [{ min: 1, max: 252 }, { min: 750, max: 758 }, { min: 840, max: 846 }, { min: 848, max: 853 }, { min: 855, max: 856 }, { min: 871, max: 871 }, { min: 893, max: 922 }, {min: 928, max: 944 }, {min: 988, max: 991}],
+  'general': [{ min: 1, max: 252 }, { min: 750, max: 758 }, { min: 840, max: 846 }, { min: 848, max: 853 }, { min: 855, max: 856 }, { min: 871, max: 871 }, { min: 893, max: 922 }, {min: 928, max: 944 }, {min: 988, max: 993}],
```

Comment:
Could you add back these examples also that were reverted.
https://github.com/microsoft/fluentui-charting-contrib/commit/7fd5f9053194538c543154c491efa2cd08b3f8bc#diff-c4c18d6aa19e941c41854fd10a557fe5aba213d31bce0c0eba88585e0b57dc25